docs/release-notes/rl-0.7.md

```diff
@@ -192,6 +192,10 @@ To migrate to `nixfmt`, simply change `vim.languages.nix.format.type` to
 - Add C# support under `vim.languages.csharp`, with support for both
   omnisharp-roslyn and csharp-language-server.
 
+- Add Julia support under `vim.languages.julia`. Note that the entirety of
+  Julia is bundled with nvf, if you enabled the module, since there is no way
+  to provide only the LSP server.
```

Comment:
This should probably be in the `vim.language.julia.lsp.enable` option description